spec/features/controllers/park_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
```

Comment:
This controller test is mis-filed as a feature test and has `:type => :request` instead of `type: controller`. Controller tests should also be organized by controller action (index, create, etc.).

Reply:
note that this test currently covers the park index action, but not the park show action

Reply:
We updated the `state_controller_spec.rb` test to include testing for the Parks index because Parks are nested in States. We've removed `park_controller_spec.rb` from this PR because there is neither a Park index nor show page to actually view, use, or test. Our serializer seemed to make this irrelevant.